### PR TITLE
display notification for first post by a user

### PR DESCRIPTION
**Pull request by kaitoo1**
### Summary of Changes
display notification for first post by a user. web + mobile

### Demo or Before/After Pics
web
CleanShot 2023-11-15 at 15 17 44@2x

mobile
CleanShot 2023-11-15 at 15 17 33
CleanShot 2023-11-15 at 15 17 53

### Edge Cases
- caption vs no caption

### Testing Steps
- with user A, follow user B who has never posted before
- with user B, create a post
- with user A, confirm you see this notification

### Checklist

Please make sure to review and check all of the following:

- [ ] I've tested the changes and all tests pass.
- [x] (if web) I've tested the changes on various desktop screen sizes to ensure responsiveness.
- [x] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

File: apps/mobile/src/components/Notification/Notifications/SomeoneYouFollowPostedTheirFirstPost.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+              <Typography
+                font={{
+                  family: 'ABCDiatype',
+                  weight: 'Bold',
+                }}
+                className="text-sm"
+              >
+                {postAuthor ? postAuthor?.username : 'Someone'}
+              </Typography>
+            </Text>
```
Comment (jakzaizzat) [inline apps/mobile/src/components/Notification/Notifications/SomeoneYouFollowPostedTheirFirstPost.tsx:90]: nit: Can we removed the `Text` tag? or is it gonna messed up the ui

File: apps/mobile/src/components/Notification/Notifications/SomeoneYouFollowPostedTheirFirstPost.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+  return (
+    <NotificationSkeleton
+      queryRef={query}
+      onPress={handlePress}
+      notificationRef={notification}
+      responsibleUserRefs={postAuthor ? [postAuthor] : []}
+    >
+      <View className="flex pt-1">
+        <View style={{ flexDirection: 'row', alignItems: 'center' }}>
```
Comment (jakzaizzat) [inline apps/mobile/src/components/Notification/Notifications/SomeoneYouFollowPostedTheirFirstPost.tsx:73]: nit: We can use tailwind class here

```
<View className="flex-row items-center"/>
```

Comment (kaitoo1) [reply apps/mobile/src/components/Notification/Notifications/SomeoneYouFollowPostedTheirFirstPost.tsx]: good call! changed

File: apps/mobile/src/components/Notification/Notifications/SomeoneYouFollowPostedTheirFirstPost.tsx (hunk 1 above)
Comment (kaitoo1) [reply apps/mobile/src/components/Notification/Notifications/SomeoneYouFollowPostedTheirFirstPost.tsx]: good call, removed!

**Review by jakzaizzat — APPROVED** [review]
LGTM!

**Pull request merged**